web-registry/src/pages/ChooseCreditClass/ChooseCreditClassPage.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { ChooseCreditClass } from '../../components/organisms/ChooseCreditClass/ChooseCreditClass';
+import { ChooseCreditClassLegacy } from '../../components/organisms/ChooseCreditClass/ChooseCreditClassLegacy';
+
+const ChooseCreditClassPage: React.FC<{ legacy?: boolean }> = ({ legacy }) => {
+  return legacy ? <ChooseCreditClassLegacy /> : <ChooseCreditClass />;
+};
+
+export { ChooseCreditClassPage };
```

Comment:
Per my earlier suggestions, I might even combine this file with `/organisms/ChooseCreditClass` - default to the new one, have the legacy in a separate file in the same folder, and return it if legacy
```tsx
if (legacy) return <CreditClassLegacy />;
return (
  ... everything that currently lives in /organisms/ChooseCreditClass
```

Reply:
I moved it all into `pages/ChooseCreditClassPage`. I agree that feels better. 

>default to the new one, have the legacy in a separate file in the same folder, and return it if legacy

I started to do this, but it feels weird because `Legacy` has a very similar (but different) structure, and 
` return legacy ? <ChooseCreditClassLegacy /> : <ChooseCreditClass />;` keeps them separate rather than giving some impression of one relying on the other (at least in my mind).

Your suggestion makes sense because once Legacy is deprecated, we can just delete that one line, but I'm gonna stick with this for now.

I like the way you are thinking about this.